apps/admin-x-framework/src/api/activitypub.ts

```diff
@@ -112,7 +112,7 @@ export const useBrowseInboxForUser = createQueryWithId<InboxResponseData>({
     headers: {
         Accept: 'application/activity+json'
     },
-    path: id => `/inbox/${id}`
```

Comment:
I think this whole file can/should be deleted - we shouldn't be using any of these requests in our app